tasking/pulp/tasking/util.py

```diff
@@ -0,0 +1,56 @@
+from contextlib import suppress
+from gettext import gettext as _
+import logging
+
+from celery import current_app, current_task
+from celery.app import control
+
+from pulp.app.models import Task
+from pulp.common import TASK_FINAL_STATES
+from pulp.exceptions import MissingResource, PulpException
+
+
+celery_controller = control.Control(app=current_app)
+_logger = logging.getLogger(__name__)
+
+
+def cancel(task_id):
+    """
+    Cancel the task that is represented by the given task_id. This method cancels only the task
```

Comment:
Can a 1-line description be added above this. I think autodoc will do better with that.